packages/regl-worldview/src/commands/Command.js

```diff
@@ -98,7 +98,7 @@ export default class Command<T> extends React.Component<Props<T>> {
     });
   }
 
-  handleMouseEvent(objects: MouseEventObject[], e: MouseEvent, ray: Ray, mouseEventName: MouseEventEnum) {
+  handleMouseEvent(objects: MouseEventObject[], ray: Ray, e: MouseEvent, mouseEventName: MouseEventEnum) {
```

Comment:
Why didn't Flow catch this at the call site?

Reply:
This file currently has flow disabled, which I didn't realize until after finishing the v0.2 release (turns out nothing has errors!). We have a ticket to fix this soon.